app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -16,7 +16,7 @@ class MainViewModel : ViewModel() {
     }
 
     // TODO: Implement the ViewModel
-    fun fetchWeatherLocations() {
+    fun fetchAllWeather() {
```

Comment:
Since this method was calling fetchAllWeather() from the service class, I went ahead and renamed it to better reflect the intended output/purpose. You could also utilize function overloading (which now that I think about, might have been something that you were going for), which is described below!

https://medium.com/coding-blocks/leveraging-the-power-of-functions-in-kotlin-de5fd9db065a

Reply:
Good catch, we probably will end up expanding on this later and have keep in mind on the naming.